README.md

```diff
@@ -1,5 +1,9 @@
 # ICESat-2 Hackweek web site
 
+*NOTE:* this website (https://icesat-2hackweek.github.io) now just redirects to icesat-2hackweek.github.io. The instructions below apply to previous version os the website, corresponding to the the 2022.03.01 tag: https://github.com/ICESAT-2HackWeek/icesat-2hackweek.github.io/tree/2022.03.01
```

Comment:
if for some reason you want to resurrect the website

Reply:
If I follow the example correctly, shouldn't this say:
```
... now just redirects to https://icesat-2.hackweek.io/
```

This is how the `index.html` meta tag is setup

Reply:
```suggestion
*NOTE:* this legacy website (https://icesat-2hackweek.github.io) automatically redirects to the newer [icesat-2hackweek.github.io](https://icesat-2.hackweek.io/). The instructions below apply to the previous version of the website, corresponding to the the 2022.03.01 tag: https://github.com/ICESAT-2HackWeek/icesat-2hackweek.github.io/tree/2022.03.01. icesat-2.hackweek.io is generated by a single, templated repository that includes all event materials.
```